docs/README.md

```diff
@@ -1,8 +1,8 @@
-# Glean
+# Glean SDK
 
 ![Glean logo](glean.jpeg)
 
-_Modern Firefox Telemetry for mobile platforms_
+_Modern telemetry for mobile platforms_
 
 The `Glean SDK` is a modern approach for a mobile Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
```

Comment:
This page is shown as the default page on the Glean SDK docs, its a bit confusing to have the Glean vs Glean-core distinction there. Can we remove all mentions to the rust part or, at least, constrain them to a separate, small section mentioning that there's a Rust WIP? In general, any chance you could address [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1559133#c1) as a follow-up? It's rather important as we're now having consumers looking at these docs.. and are confused :)